app/models/concerns/orchestration/tftp.rb

```diff
@@ -145,19 +149,19 @@ def queue_tftp_update
     if mac != old.mac
       set_tftp = true
       # clean up old TFTP reservation file
-      if old.tftp?
-        queue.create(:name => _("Remove old TFTP Settings for %s") % old, :priority => 19,
-                     :action => [old, :delTFTP])
-      end
+      queue_tftp_destroy(false) if old.tftp?
```

Comment:
This now has a priority of 20 instead of 19 and does not use `old`, but `self`. Why change the behavior?